pom.xml

```diff
@@ -99,13 +99,7 @@
             <!-- Required for running with Java 9+ -->
             <groupId>org.jboss.marshalling</groupId>
             <artifactId>jboss-marshalling-river</artifactId>
-            <version>2.0.5.Final</version>
-        </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci</groupId>
-            <artifactId>annotation-indexer</artifactId>
```

Comment:
`mvn dependency:tree` shows that we already pick up `org.jenkins-ci:annotation-indexer:jar:1.12:provided` through core, so better to remove this.

Reply:
I do understand the point, but I disagree. If you use a library, you should say so in your `pom.xml`. See `mvn dependency:analyze`, you will have "undeclared" libraries. The point is, if you don't list the dependency, and the other one which was providing it by transitivity stop using it (won't happen with core, yes), then you might have issues during an upgrade.

Reply:
Yes, my opinion would be different if the library was not in core, but depending on a specific version of the library here just opens us up to accidental version mismatches with core which could be problematic.